api/desecapi/pdns_change_tracker.py

```diff
@@ -5,7 +5,7 @@
 from django.db.transaction import atomic
 from django.utils import timezone
 
-from desecapi import metrics
+from desecapi import metrics, pch
 from desecapi.models import RRset, RR, Domain
 from desecapi.pdns import (
```

Comment:
`from desecapi import pdns`

Reply:
done & refactored to avoid usage of private API and for better separation of concerns